### PR TITLE
doc/releases/octopus.rst: add release notes for 15.2.8

### PR DESCRIPTION
**Pull request by neha-ojha**
Signed-off-by: Neha Ojha <nojha@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by batrick** [comment]
````
-- Checking for one of the modules 'yaml-cpp'
CMake Error at /usr/share/cmake-3.10/Modules/FindPackageHandleStandardArgs.cmake:137 (message):
  Could NOT find yaml-cpp: Found unsuitable version "", but required is at
  least "0.5.1" (found yaml-cpp_LIBRARY-NOTFOUND)
Call Stack (most recent call first):
  /usr/share/cmake-3.10/Modules/FindPackageHandleStandardArgs.cmake:376 (_FPHSA_FAILURE_MESSAGE)
  cmake/modules/Findyaml-cpp.cmake:46 (find_package_handle_standard_args)
  src/CMakeLists.txt:331 (_find_package)
  src/seastar/cmake/SeastarDependencies.cmake:98 (find_package)
  src/seastar/CMakeLists.txt:330 (seastar_find_dependencies)


-- Configuring incomplete, errors occurred!
See also "/home/jenkins-build/build/workspace/ceph-pull-requests/build/CMakeFiles/CMakeOutput.log".
See also "/home/jenkins-build/build/workspace/ceph-pull-requests/build/CMakeFiles/CMakeError.log".
+ exit 1
````

**Comment by batrick** [comment]
jenkins test make check

**Comment by neha-ojha** [comment]
> ```
> -- Checking for one of the modules 'yaml-cpp'
> CMake Error at /usr/share/cmake-3.10/Modules/FindPackageHandleStandardArgs.cmake:137 (message):
>   Could NOT find yaml-cpp: Found unsuitable version "", but required is at
>   least "0.5.1" (found yaml-cpp_LIBRARY-NOTFOUND)
> Call Stack (most recent call first):
>   /usr/share/cmake-3.10/Modules/FindPackageHandleStandardArgs.cmake:376 (_FPHSA_FAILURE_MESSAGE)
>   cmake/modules/Findyaml-cpp.cmake:46 (find_package_handle_standard_args)
>   src/CMakeLists.txt:331 (_find_package)
>   src/seastar/cmake/SeastarDependencies.cmake:98 (find_package)
>   src/seastar/CMakeLists.txt:330 (seastar_find_dependencies)
> 
> 
> -- Configuring incomplete, errors occurred!
> See also "/home/jenkins-build/build/workspace/ceph-pull-requests/build/CMakeFiles/CMakeOutput.log".
> See also "/home/jenkins-build/build/workspace/ceph-pull-requests/build/CMakeFiles/CMakeError.log".
> + exit 1
> ```

not related, being addressed in https://github.com/ceph/ceph/pull/38609

**Review by batrick — APPROVED** [review]

File: doc/releases/octopus.rst [hunk 1]
```diff
@@ -1,3 +1,233 @@
+v15.2.8 Octopus
+===============
+
+This is the 8th backport release in the Octopus series. This release fixes
+a security flaw in CephFS. We recommend users to update to this release.
```
Comment (jdurgin) [inline doc/releases/octopus.rst:5]: and includes a number of bug fixes

Comment (neha-ojha) [reply doc/releases/octopus.rst]: added

**Review by jdurgin — APPROVED** [review]

**Comment by batrick** [comment]
jenkins test make check

**Comment by neha-ojha** [comment]
jenkins test make check

**Pull request merged**